ckanext/switzerland/helpers/plugin_utils.py

```diff
@@ -100,10 +100,7 @@ def ogdch_prepare_search_data_for_index(search_data, format_mapping):  # noqa
     search_data['groups_fr'] = [ogdch_loc_utils.get_localized_value_from_dict(g['display_name'], 'fr') for g in validated_dict[u'groups']]  # noqa
     search_data['groups_it'] = [ogdch_loc_utils.get_localized_value_from_dict(g['display_name'], 'it') for g in validated_dict[u'groups']]  # noqa
     search_data['res_description'] = [ogdch_loc_utils.lang_to_string(r, 'description') for r in validated_dict[u'resources']]  # noqa
-    search_data['res_format'] = ogdch_format_utils.prepare_formats_for_index(
-        resources=validated_dict[u'resources'],
-        format_mapping=format_mapping
-    )  # noqa
+    search_data['res_format'] = ogdch_format_utils.prepare_formats_for_index(resources=validated_dict[u'resources'])  # noqa
```

Comment:
Just the function call is simplified because ``format_mapping`` is derived in the module and does not be passed on as an argument.